index.html

```diff
@@ -0,0 +1,554 @@
+                    <p><span>COFFEE EVERYWHERE</span> Now turn around and look a mile in the other direction. You see a
+                        similar coffee cup, at exactly the same temperature. Coincidence? Maybe. But if you see a
+                        similar
+                        cup in every direction, you might want to look for another explanation.</p>
+                </div>
+            </div>
+            <div class="second">
+                <div class="card">
+                    <div class="image">
+                        <img src="./Images/Screenshot_5.jpg" alt="">
+                    </div>
+                    <p><span>STILL NOT ENOUGH TIME</span> There has not been enough time to carry coffee cups from place
+                        to
+                        place before they get cold. But if all the coffee cups were somehow filled from a single coffee
+                        pot,
+                        all at the same time, that might explain their even temperature.</p>
+                </div>
+                <div class="card">
+                    <div class="image">
+                        <img src="./Images/Screenshot_6.jpg" alt="">
+                    </div>
+                    <p><span>INFLATION</span> solves this problem. The theory proposes that, less than a trillionth of a
+                        second after the Big Bang, the universe expanded faster than the speed of light. Tiny ripples in
+                        the
+                        violently expanding energy field eventually grew into the large-scale structures of the
+                        universe.
+                    </p>
+                </div>
+                <div class="card">
+                    <div class="image">
+                        <img src="./Images/Screenshot_7.jpg" alt="">
+                    </div>
+                    <p><span>FLUCTUATION</span> Astronomers have now detected evidence of these ancient fluctuations in
+                        swirls of polarized light in the cosmic background radiation, which is energy left over from the
+                        early universe. These are gravitational waves predicted by Einstein.</p>
+                </div>
+                <div class="card">
+                    <div class="image">
+                        <img src="./Images/Screenshot_8.jpg" alt="">
+                    </div>
+                    <p><span>EXPANSION </span> Returning to our coffee, imagine a single, central pot expanding faster
+                        than
+                        light and cooling to an even temperature as it expands. That is something like inflation. And
+                        the
+                        structure of the universe mirrors the froth and foam of the original pot.</p>
+                </div>
+            </div>
+        </div>
+        <hr>
+        <p class="larry">By LARRY BUCHANAN and JONATHAN CORUM</p>
+    </div>
```

Comment:
- [ ] Do not use screenshots of the images, instead use one image just like the website (use the `position` property) ([link to page](https://www.nytimes.com/2014/03/18/science/space/detection-of-waves-in-space-buttresses-landmark-theory-of-big-bang.html?_r=0))